agent/functional_tests/README.md

```diff
@@ -28,4 +28,6 @@ agent and access the ECS APIs.
 
 The best way to run them is via the `make run-functional-tests` target.
 
-Thay may also be manually run with `go test -tags functional -v ./...`,
+Thay may also be manually run with `go test -tags functional -v ./...`.
```

Comment:
Nit, before running them this way you typically also want to `source ./scripts/shared_env` once to get dependencies on your `GOPATH`